src/Microsoft.Data.Analysis/ColumnArithmeticTemplate.ttinclude

```diff
@@ -88,6 +120,21 @@
         return false;
     }
 
+    // These are a subset of the implicit conversions allowed in C#. They are used to generate the return type for binary ops
```

Comment:
It might be good to add a reference to the "spec" here: [https://github.com/dotnet/csharplang/blob/master/spec/conversions.md#implicit-numeric-conversions](https://github.com/dotnet/csharplang/blob/master/spec/conversions.md#implicit-numeric-conversions)